src/components/Option.vue

```diff
@@ -1,19 +1,24 @@
 <template>
-  <div
+  <button
     class="vcs__option"
     :class="{
       'vcs__option--active': active,
       'vcs__option--disabled': disabled,
+      'vsc__option--focused': focused,
     }"
-    @mouseenter="$emit('openMenu', value, options)"
+    @mouseenter="$emit('openMenu', value, options, index, false)"
     @click="handleSelection"
+    @keydown.right="$emit('openMenu', value, options, index, true)"
+    @keydown.left="$emit('closeMenu')"
+    @keydown.down="$emit('nextOption')"
+    @keydown.up="$emit('prevOption')"
```

Comment:
Are these events emitted just when the `Option` is focused or at all times?

Does `@keydown.left` close all menus or just the last one? I think it should only close the last one.

Reply:
Hey, they are triggered only on the focused option. And yes, it only closes the last menu